toolkit/scripts/containerized-build/resources/splash.md

```diff
@@ -0,0 +1,3 @@
+# Generate splashes with:
+figlet "Mariner Builder!" |lolcat --seed 0  --force > ./resources/builder_splash.txt && cat ./resources/builder_splash.txt
+figlet "Mariner Tester!" | lolcat --seed 0  --force > ./resources/tester_splash.txt && cat ./resources/tester_splash.txt
```

Comment:
Any reason you are precreating the splash text instead of creating it on the fly when building the container?

Reply:
Not a great reason, but the tool would be using either of the splashes for sure, so maybe makes sense to just have it?

Reply:
Consider dynamically generating the splash texts so we don't get inadvertent hits when grepping through the source tree

Reply:
Generating this dynamically would require figlet and lolcat (or some other package) be installed on the user's system. Not sure if that's a fair ask from the user.
If we have a way to generate banner with native linux command, we could try that.
Alternatively, we can get rid of the splash banner altogether.

Reply:
For reference, Ubuntu, Fedora, and Arch all have in-built packages for `figlet` and `lolcat`. So, I don't think such a requirement would be too problematic.

That being said, they are missing from Mariner, which is obviously a great travesty that must be rectified immediately. 😉